examples/test_odx/test_server_odx.py

```diff
@@ -0,0 +1,73 @@
+import logging
```

Comment:
add the BOSCH header and module explanation

Reply:
added